index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap">
+        <title>Portfolio</title>
+</head>
+
+<body>
+  
+  <header>
+    <nav title="navigation-bar">
+      <button class="nav-button1">
+        <span class="logo">Joy</span>
```

Comment:
- [ ] Please make this logo a link tag that navigates back to the home page. This is for three reasons: convention, semantics, and user experience(UX). 
[Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.😃